cwl/.reana.yaml

```diff
@@ -0,0 +1,11 @@
+metadata:
```

Comment:
https://github.com/reanahub/reana-demo-root6-roofit/blob/master/.reana.yaml this is the last REANA spec so it works with last `reana-client` (the only thing that will change soon is that `parameters` will be moved under `inputs:`).